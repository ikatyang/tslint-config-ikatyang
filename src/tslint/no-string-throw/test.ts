// good

function good1() {
  throw new Error('message');
}

// bad

function bad1() {
  throw 'message';
}
