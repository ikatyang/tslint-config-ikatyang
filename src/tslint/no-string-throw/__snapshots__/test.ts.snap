// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`lint 1`] = `
// good

function good1() {
  throw new Error('message');
}

// bad

function bad1() {
  throw 'message';
  ~~~~~~~~~~~~~~~~ [Throwing plain strings (not instances of Error) gives no stack traces]
}

`;
